docs/posts/run_latest_gnome_kde_on_distrobox.md

```diff
@@ -92,6 +94,7 @@ X-GDM-SessionRegisters=true
 ```
 
 This file should be placed under `/usr/local/share/wayland-sessions/distrobox-gnome.desktop`
+(If it doesn't show up, you can place it under `/usr/share/xsessions/distrobox-gnome.desktop`)
```

Comment:
Hmm, may I guess - you have SDDM on host?

Reply:
@damianoognissanti then if it's GDM it should be no problem to put it by `/usr/local/...` path.
Did you try to investigate why it's not for your case?

Reply:
@bam80 Yeah, I know, that was very odd. Since it worked with this fix I let it go and thought the comment could help someone where the standard path wouldn't work.

Reply:
Maybe your GDM runs on X11 and thus doesn't pick up the Wayland session files?